api/image/image_test.go

```diff
@@ -64,9 +76,27 @@ func TestSplit(t *testing.T) {
 		},
 		{
 			testName: "with digest",
-			value:    "nginx@12345",
+			value:    "nginx@sha256:12345",
 			name:     "nginx",
-			tag:      "@12345",
+			tag:      "@sha256:12345",
```

Comment:
Aligned with the above suggestion to change the `Split` function to return the tag and digest separately, I'd like to see this test look like the following:

```
			testName: "with digest",
			value:    "nginx@sha256:12345",
			name:     "nginx",
			digest:   "sha256:12345"
```